### PR TITLE
Add new symfony4 site type to homestead documentation

### PR DESCRIPTION
**Pull request by michaelesmith**
A new site type was added in https://github.com/laravel/homestead/pull/600 for index.php only configurations. This pr adds this information to the homestead documentation.

### CODE REVIEW

File: setup/homestead.rst [hunk 1]
```diff
@@ -50,9 +50,11 @@ configuration:
     sites:
         - map: symfony-demo.dev
           to: /home/vagrant/projects/symfony_demo/web
-          type: symfony
+          type: symfony4
 
-The ``type`` option tells Homestead to use the Symfony nginx configuration.
+The ``type`` option tells Homestead to use the Symfony nginx configuration. 
+Homestead now supports a Symfony2 web layout with app.php and app_dev.php when
+using type: symfony2 and an index.php layout when using type: symfony4.
```
Comment (xabbuh) [inline setup/homestead.rst]: What is happening when the `symfony` layout (with neither 2 nor 4) is used as it was before?

Comment (michaelesmith) [inline setup/homestead.rst]: When `symfony` is used it defaults to `serve_symfony2.sh` configuration (this hasn't changed). I added the symfony2 for clarity when discussing the new `symfony4` type.

**Review by javiereguiluz — CHANGES_REQUESTED** [review]
The front controller of Symfony 4 has changed. Instead of `web/index.php`, it'll be `public/index.php`

**Review by xabbuh — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]

**Comment by wouterj** [comment]
Hi @michaelesmith!

I've rebased your PR on the 4.0 version of the docs and added some minor inline code blocks in a seperate commit. This is now merged with PR https://github.com/symfony/symfony-docs/pull/8640. You'll still receive your credits for your nice work! Thanks for documenting this new homestead feature.

**Pull request closed without merging**